File: BUSAPP/src/java/modelos/Bus.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package modelos;

/**
 *
 * @author JORGE_ALEJANDRO
 */
public class Bus {
    private int id;
    private String placa;
    private String nombreConductor;
    private String tipo;
    private int valorPasaje;

    public Bus(int id, String placa, String nombreConductor, String tipo, int valorPasaje) {
        this.id = id;
        this.placa = placa;
        this.nombreConductor = nombreConductor;
        this.tipo = tipo;
        this.valorPasaje = valorPasaje;
    }

    

    public int getId() {
        return id;
    }

    public String getNombreConductor() {
        return nombreConductor;
    }

    public String getTipo() {
        return tipo;
    }

    public String getPlaca() {
        return placa;
    }

    public int getValorPasaje() {
        return valorPasaje;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setNombreConductor(String nombreConductor) {
        this.nombreConductor = nombreConductor;
    }

    public void setTipo(String tipo) {
        this.tipo = tipo;
    }
    
    

    public void setPlaca(String placa) {
        this.placa = placa;
    }

    public void setValorPasaje(int valorPasaje) {
        this.valorPasaje = valorPasaje;
    }
    
    
    
}

File: BUSAPP/web/scripts/bus.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
$(document).ready(function () {
//        $("registrarBus").click(registrarBus);
//        function registrarBus()
//        {
//            var placa = $("#placa").val();
//            var nombreConductor = $("#nombreConductor").val();
//            var tipo = $("#tipo").val();
//            var valorPasaje = $("#valorPasaje").val();
//
//            if (placa != null && nombreConductor != null && tipo != null && valorPasaje != null)
//            {
//                $.post("Controladora",
//                        {
//                            operacion: "registrarBus",
//                            placa: placa,
//                            nombreConductor: nombreConductor,
//                            tipo: tipo,
//                            valorPasaje: valorPasaje
//                        },
//                        function (data)
//                        {
//                            if (data)
//                            {
//                                alert("Se registro correctamente el bus");
//                            } else
//                            {
//                                alert("Erro al registrar bus");
//                            }
//                        }, "json");
//            } else
//            {
//                alert("Existe algun campo vacio")
//            }
//        }

    $.post("Controladora1",
            {
                operacion: "mostrarBuses",
            },
            function (data)
            {
                if (data != null)
                {
                    var tablaBuses = $("#tablaBuses");
                    for (var i = 0; i < data.length; i++) {
                        var idBus = data[i].id;
                        var placa = data[i].placa;
                        var nombreConductor = data[i].nombreConductor;
                        var tipo = data[i].tipo;
                        var valorPasaje = data[i].valorPasaje;
                        var tr = $("<tr>");
                        var td1 = $("<td>", {text: idBus});
                        var td2 = $("<td>", {text: placa});
                        var td3 = $("<td>", {text: nombreConductor});
                        var td4 = $("<td>", {text: tipo});
                        var td5 = $("<td>", {text: valorPasaje});

                        tr.append(td1);
                        tr.append(td2);
                        tr.append(td3);
                        tr.append(td4);
                        tr.append(td5);
                        tablaBuses.append(tr);
                    }
                } else
                {
                    alert("Error al mostrar buses");
                }
            }, "json");


}
);

